commcare_connect/opportunity/migrations/0045_alter_opportunity_auto_approve_payments_and_more.py

```diff
@@ -0,0 +1,29 @@
+# Generated by Django 4.2.5 on 2024-05-30 11:14
+
+from django.db import migrations, models
+
+
+def change_defaults(apps, schema_editor):
```

Comment:
This isn't changing the defaults, it changes everything right? Is that intentional? We want to change how existing projects work? If so, we should change the name of this method, but that seems overly aggressive since some of these projects are in progress.

Reply:
(sorry for the delay here).
> We want to change how existing projects work? 
Per my chat with Rama earlier, this seems what he wanted.

[3:33 PM](https://dimagi.slack.com/archives/D057T7G55DJ/p1717063383265839)
Sravan Reddy
 Should we turn Auto Approve Visits on for all existing opportunities (even the ones that have them OFF)?

3:33 PM
And for ‘Auto Approve Payments’ we don’t change for existing opportunities?
3:34 PM
Rama Siva
:palm_tree: You mean for existing Opportunities. Yes. There is currently only 1 live Opportunity that is real. They have already turned it on for both.
[3:34 PM](https://dimagi.slack.com/archives/D057T7G55DJ/p1717063445057319)
So no need to worry about what will happen to existing opportunities. Make it the same for past, current and future opportunities.